kieserver/modules/kieserver/module.yaml

```diff
@@ -29,16 +29,16 @@ envs:
 artifacts:
 - name: "rhpam_kie_server_distribution.zip"
   # rhpam-7.11.0-kie-server-ee8.zip
-  md5: "e5353118cbd2e739f46885cda6d133cd"
+  md5: "da8ff89813f43a329d5bd2e863011308"
 - name: "rhpam_business_central_distribution.zip"
   # rhpam-7.11.0-business-central-eap7-deployable.zip
-  md5: "ecfa904ecdc416b90014a9b0629d8e86"
+  md5: "c0273ba50552f9c13631da8fcc1ee39a"
 - name: "slf4j-simple.jar"
   # slf4j-simple-1.7.22.redhat-2.jar
   md5: "62cc6eeb72e2738e3acc8957ca95f37b"
 - name: "kie-server-services-jbpm-cluster-7.52.0.Final-redhat-00004.jar"
```

Comment:
This has to be
```
name: "kie-server-services-jbpm-cluster-7.52.0.Final-redhat-00007.jar"
md5: "139aeae58d08eda539a1c72778967b3a"
```

Reply:
I will raise a seperate PR for it just after this PR merged.